modules/vba_documents/app/workers/vba_documents/upload_processor.rb

```diff
@@ -72,7 +72,6 @@ def download_and_process
         metadata = perfect_metadata(@upload, parts, timestamp)
 
         pdf_validator_options = VBADocuments::DocumentRequestValidator.pdf_validator_options
-        pdf_validator_options[:check_page_dimensions] = false if metadata['skipDimensionCheck'].present?
```

Comment:
I've confirmed with Kyle that it's ok for us to remove this behavior as long as including the metadata key doesn't start causing errors